### PR TITLE
feat(astro): allow --no-integrations via cli

### PR DESCRIPTION
**Pull request by sergeylukin**
## What it does

Allows running generator with --no-integrations without failing

### CODE REVIEW

**Comment by leosvelperez** [comment]
Thanks for contributing!

Please note the `integrations` flag is not a `boolean`. It's an `array` and it defaults to an empty array. Passing `--no-integrations` is not something you should do or need to do.

Still, the change of this PR shouldn't make a difference. The `GeneratorCallback` task for the integrations is only added to the `tasks` array if defined as you can see in https://github.com/nxtensions/nxtensions/blob/main/packages/astro/src/generators/application/generator.ts#L36-L38. Same is done in the library generator.

I'm closing this. If you still have issues with this, please create an issue first with a reproduction so we can discuss it and troubleshoot it.

**Comment by sergeylukin** [comment]
Hey! Thanks for expanded reply and for your time. Apparently, I was in a rush when submitted this PR and have mistyped the name of the flag in subject which took this discussion off the rails. Sorry about that. Please let me rephrase :) This PR resolves problem which occurs when @nxtensions/astro is used to generate an app with `--no-interactive` flag, please find the bug reproduction and bug fix scenarios below.

## Reproduction

Just run this CLI snippet:

```sh
npx create-nx-workspace@latest nx-astro --preset=empty --nxCloud=false --pm=npm && \
cd nx-astro && \
npm i -D @nxtensions/astro && \
npx nx g @nxtensions/astro:app website --no-interactive
```

Screen Shot 2023-03-24 at 0 15 37

## Fix

...and here is the same scenario with this PR fix included (installing @nxtensions/astro from my fork with this PR's change):

```
npx create-nx-workspace@latest nx-astro --preset=empty --nxCloud=false --pm=npm && \
cd nx-astro && \
npm i -D "@nxtensions/astro@sergeylukin/nxtensions#no-integrations" && \
npx nx g @nxtensions/astro:app website --no-interactive
```

Screen Shot 2023-03-24 at 0 45 03

Hope that makes sense!

P.S. if you know a way to install npm package from github **mono**repo, LMK :) for now I had to create a [special branch](https://github.com/sergeylukin/nxtensions/tree/no-integrations) which contains package in the root folder in addition to the [normal branch with the change](https://github.com/sergeylukin/nxtensions/tree/allow-no-integrations-via-cli) which I couldn't get working with npm due to monorepo structure.

**Comment by leosvelperez** [comment]
Oh, I see. I reproduced it now and the issue happens because in a recent Nx version the`cypressInitGenerator` changed to return a Promise. That was fixed in https://github.com/nxtensions/nxtensions/pull/335/files#diff-11a83b544887ceef518edf5976ebe10f8348e3c687853651706d06823d2b2be3R38, but I just realized I hadn't released the changes. I'll create a release now and that should address this issue.
